<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mymusic</class>
 <widget class="QWidget" name="Mymusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	border-image: url(:/pictur/bg_music1.jpg);
}
#frame_2{
	background-color: rgba(0, 0, 0, 80);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	background-color: rgba(255, 255, 255, 100);
	color: rgb(0, 0, 0);
border: 2px solid rgb(153, 230, 230);
border-radius:5px;
margin-top:2.1ex;
font-family:宋体;
font:bold 30px;
color: ;
	color: rgb(0, 247, 119);
}
 
QGroupBox::title
{
subcontrol-origin:margin;
subcontrol-position:top center;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>560</y>
       <width>1001</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 5px;
	background-color: rgba(255, 255, 255,200);

font-size: 15px;
font-weight: bold;
color: rgb(130, 130, 130);


}</string>
     </property>
     <property name="title">
      <string>控制</string>
     </property>
     <widget class="QPushButton" name="pushButton_pre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>上一曲</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_playORpause">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_next">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>下一曲</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_list">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>单曲循环</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>顺序播放</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>列表循环</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>随机播放</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>音量</string>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>30</y>
        <width>22</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {
border: 1px solid #4A708B;
background: #C0C0C0;
width: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
padding-top:-1px;
padding-bottom:-1px;
}

QSlider::sub-page:vertical {
background: #575757;
border: 1px solid #4A708B;
border-radius: 2px;
}

QSlider::add-page:vertical {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #c4c4c4, stop:1 #B1B1B1);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #5DCCFF, stop: 1 #1874CD);
border: 0px solid #777;
width: 10px;
border-radius: 2px;
}

QSlider::handle:vertical 
{

    height: 11px;
    margin-left: -3px;
    margin-right: -3px;
    border-radius: 5px;
}

QSlider::sub-page:vertical:disabled {
background: #00009C;
border-color: #999;
}

QSlider::add-page:vertical:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:vertical:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="Slider_time">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*horizontal ：水平QSlider*/
QSlider::groove:horizontal {
border: 0px solid #bbb;
}

/*1.滑动过的槽设计参数*/
QSlider::sub-page:horizontal {
 /*槽颜色*/
background: rgb(90,49,255);
 /*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:8px;
 /*下遮住区域高度*/
margin-bottom:8px;
/*width在这里无效，不写即可*/
}

/*2.未滑动过的槽设计参数*/
QSlider::add-page:horizontal {
/*槽颜色*/
background: rgb(255,255, 255);
/*外环大小0px就是不显示，默认也是0*/
border: 0px solid #777;
/*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:9px;
 /*下遮住区域高度*/
margin-bottom:9px;
}
 
/*3.平时滑动的滑块设计参数*/
QSlider::handle:horizontal {
/*滑块颜色*/
background: rgb(193,204,208);
/*滑块的宽度*/
width: 5px;
/*滑块外环为1px，再加颜色*/
border: 1px solid rgb(193,204,208);
 /*滑块外环倒圆角度*/
border-radius: 2px; 
 /*上遮住区域高度*/
margin-top:6px;
 /*下遮住区域高度*/
margin-bottom:6px;
}

/*4.手动拉动时显示的滑块设计参数*/
QSlider::handle:horizontal:hover {
/*滑块颜色*/
background: rgb(193,204,208);
/*滑块的宽度*/
width: 10px;
/*滑块外环为1px，再加颜色*/
border: 1px solid rgb(193,204,208);
 /*滑块外环倒圆角度*/
border-radius: 5px; 
 /*上遮住区域高度*/
margin-top:4px;
 /*下遮住区域高度*/
margin-bottom:4px;
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_currentTime">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>110</y>
       <width>250</width>
       <height>391</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>播放列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_add">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_del">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_gif">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>361</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/pictur/music.gif);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>481</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>搜索</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_find">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
</string>
        </property>
        <property name="placeholderText">
         <string>输入歌名</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
